.github/workflows/zephyr.yml

```diff
@@ -20,5 +20,5 @@ jobs:
 
       - name: build
         run: docker run -v "$(pwd)":/workdir
-             docker.io/zephyrprojectrtos/zephyr-build:latest
+             docker.io/zephyrprojectrtos/zephyr-build:v0.17.3
```

Comment:
If we set the version, we need to update occasionally if docker image is updated.
It looks like zephyr's latest image is not stable. Why is so? Can they keep different tag until it get stabilized?

Reply:
Very good question. I've asked the image maintainers on the #infrastructure channel athttps://zephyrproject.slack.com/archives/CGNR3LCHM about a more `:stable` version but there is none. 

Quoting the same channel:
 > right now we might have a few updates as we are catching up after we merged the 2 dockers, but usually it should be less frequent

I believe we will NOT have to update for every single image release. For now this is just a build test so if it builds with a non-latest image then it is good enough and if it breaks then we update the image (hence the Zephyr SDK) "on-demand". We don't want to force developers to constantly update their toolchain either after all.

We already build Zephyr with the random, latest Zephyr commit of the day so maybe it's good to have one less moving part, should help point the finger when something outside of SOF breaks this.